template.yml

```diff
@@ -5,6 +5,16 @@ Description: >
   The resources for providing a service for listing users and roles.
   It includes a table containing users and roles and lambda functions for accessing the data.
 
+Metadata:
```

Comment:
Is this something we should add to all applications?

Reply:
All applications that will be published to SAR, yes